src/glwidget.cpp

```diff
@@ -34,16 +35,13 @@
 #include <QtEvents>
 #include "post_guard.h"
 
-#if QT_VERSION >= 0x040800
-    #ifdef __APPLE__
-        #include <OpenGL/glu.h>
-    #else
-        #include <GL/glu.h>
-    #endif
-#endif
-
 #include <math.h>
 
+#ifdef __APPLE__
```

Comment:
This seems curiously out of place and inconsistent - ought it to be `QT_OS_MACOS` or whatever?